src/Nox.ClientApi.Tests/Generated/Nox.Generator/Nox.Generator.NoxCodeGenerator/Presentation.Api.OData/WorkplacesController.g.cs

```diff
@@ -184,6 +207,30 @@ public async Task<ActionResult> DeleteRefToTenants([FromRoute] System.UInt32 key
         return NoContent();
     }
     
+    [HttpPut("api/Workplaces/{key}/Tenants/{relatedKey}")]
+    public virtual async Task<ActionResult<TenantDto>> PutToTenantsNonConventional(System.UInt32 key, System.Guid relatedKey, [FromBody] TenantUpdateDto tenant)
+    {
+        if (!ModelState.IsValid)
+        {
+            return BadRequest(ModelState);
+        }
+        
+        var related = (await _mediator.Send(new GetWorkplaceByIdQuery(_cultureCode, key))).Select(x => x.Tenants).SingleOrDefault()?.SingleOrDefault(x => x.Id == relatedKey);
```

Comment:
Would this work, or similiar to?
```suggestion
        var related = (await _mediator.Send(new GetWorkplaceByIdQuery(_cultureCode, key))).Select(x => x.Tenants).SingleOrDefault()?.Any(x => x.Id == relatedKey);
```

Reply:
This one is updated